R/summarize_copilot.R

```diff
@@ -299,12 +301,20 @@ map_responses_to_cycles <- function(response_tbl,
   # * created_date
   # * cycle_ordinal
 
+  require(sqldf)
+  require(utils)
```

Comment:
Are these `require()` steps necessary given `modules::import()` above? If not, are they likely to cause problems?

Reply:
Got it! Thanks